test/tools/jsontests/BlockChainTests.cpp

```diff
@@ -104,144 +105,187 @@ void checkBlocks(TestBlock const& _blockFromFields, TestBlock const& _blockFromR
 bigint calculateMiningReward(u256 const& _blNumber, u256 const& _unNumber1, u256 const& _unNumber2, SealEngineFace const& _sealEngine);
 json_spirit::mObject fillBCTest(json_spirit::mObject const& _input);
 void testBCTest(json_spirit::mObject const& _o);
+void spellCheckNetworkNamesInExpectField(json_spirit::mArray const& _expects);
 
-//percent output for many tests in one file
-json_spirit::mValue doBlockchainTests(json_spirit::mValue const& _v, bool _fillin)
-{
-	TestOutputHelper testOutputHelper(_v.get_obj().size());	//Count how many tests in the json object (read from .json file)
-	json_spirit::mValue ret = doBlockchainTestNoLog(_v, _fillin); //Do the test / test generation
-	return ret;
-}
 
-json_spirit::mValue doTransitionTest(json_spirit::mValue const& _input, bool _fillin)
+class BlockchainTestSuite: public TestSuite
```

Comment:
I think it makes sense to create `BlockChainTests.h` and declare the class there.

Reply:
there is no way to declare a virtual function override and separate it from actual realisation

Reply:
there is of course. don't write `override` in implementation, but leave it in the header